en/micro-integrator/docs/setup/deployment/databases/setting-up-MySQL.md

```diff
@@ -0,0 +1,56 @@
+
+# Configuration a MySQL database
+
+Given below are the steps you need to follow in order to use a MySQL database for cluster coordination in a Micro Integrator cluster.
+
+## Setting up MySQL
+
+Following the steps below to create the necessary databases.
+
+1. Download and install [MySQL Server](http://dev.mysql.com/downloads/).
+2. Download the [MySQL JDBC driver](http://dev.mysql.com/downloads/connector/j/).
+3. [Download](http://wso2.com/integration) and unzip the database scripts for the Micro Integrator.
+4. Unzip the downloaded MySQL driver and copy the MySQL JDBC driver JAR (mysql-connector-java-x.x.xx-bin.jar) into the `MI_HOME/lib/` directory of your Micro Integrator.
+5. Execute the following command in a terminal/command where the username is the username you want to use to access the databases:
+
+	 `mysql -u username -p`
+
+6. When prompted, specify the password to access the databases with the username you specified.
+
+## Creating the database
+
+Create the user DB by pointing to the MySQL script that you downloaded. Execute the following command:
+
+!!! Tip
+	If you are using MySQL 5.7 or a later version, you need to use the mysql5.7.sql script instead of the mysql.sql script. This script has been tested on MySQL 5.7 and MySQL 8.
```

Comment:
this is not applicable to us , we have only one script . Also the script names are different .